recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: yaml-{{ version }}.tar.gz
   url: http://pyyaml.org/download/libyaml/yaml-{{ version }}.tar.gz
-  md5: 5fe00cda18ca5daeb43762b80c38e06e
+  md5: 1abf45bd3a96374fa55ca63b32f9f2f9
```

Comment:
Could we please switch to `sha256`? Admittedly it was this way before your update, but it would be a better check to use.